src/RouteLayout.ts

```diff
@@ -17,14 +17,12 @@ export function setupLayouts(routes) {
       if (route.children?.length > 0) {
         route.children = deepSetupLayout(route.children, false)
       }
-
-      if (!route.component) return route
       
       if (top && route.meta?.layout !== false) {
         return { 
           path: route.path,
           component: layouts[route.meta?.layout || '${options.defaultLayout}'],
-          children: route.path === '/' ? [route] : [{...route, path: ''}],
```

Comment:
Why change the code here? This is a special patch.

https://github.com/JohnCampionJr/vite-plugin-vue-layouts/pull/97

Now it's a problem again.

Reply:
I apologize for overlooking this case.
I will revert this change while maintaining compatibility with solution #130

Reply:
done → #137